blast/formatter/server/internal/server/run-report-ep.go

```diff
@@ -100,11 +112,21 @@ func zipDir(path string) (err error) {
 
 	var tf io.Writer
 	for i := range matches {
-		tf, err = zp.Create(filepath.Base(matches[i]))
+
+		baseFileName := filepath.Base(matches[i])
+
+		// This should never happen given the check above, but just to be safe!
+		if baseFileName == reportExportName {
+			return errors.New(baseFileName + " already exists!")
+		}
+
+		// Create a file reference in the new zip for the globbed file
+		tf, err = zp.Create(baseFileName)
```

Comment:
Additional check (for race condition reasons) to ensure the zip file is not in the globbed file set.